bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/proxy/ProxyServletService.java

```diff
@@ -96,6 +113,28 @@
     protected ItemUIRegistry itemUIRegistry;
     protected ModelRepository modelRepository;
```

Comment:
The `modelRepository` does not seem to be used anymore, so I'd say it should be removed.

Reply:
Leftover from testing.   Removed.

Reply:
And I was wrong, it is used.